spec/factories/comments.rb

```diff
@@ -0,0 +1,9 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :comment do
+    body 'Most interresting comment ever, created by a girl.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.